samples/Basic-Bot/Startup.cs

```diff
@@ -123,10 +123,15 @@ public void ConfigureServices(IServiceCollection services)
             var entityStateAccessor = userState.CreateProperty<Dictionary<string, object>>("EntityState");
             services.AddSingleton(entityStateAccessor);
 
+            // The application values must match the ones during creation of the endpoint
             var applicationId = "lgshowcases";
-            var endpointKey = Keys.LanguageGenerationSubscriptionKey;
-            var endpointRegion = "westus"; // The region must be the subscription key's region.
-            var languageGenerationMiddleware = LanguageGenerationUtilities.CreateMiddleware(applicationId, endpointKey, endpointRegion, entityStateAccessor);
+            var applicationRegion = "westus";
```

Comment:
Should those be constant somewhere else?

Reply:
No, as these values should be explicitly set by the developer using the samples.